### DISCUSSION TITLE
[Feature] Create album from selection

### DISCUSSION THREAD

**Discussion by erikvanoosten** [immich-app/immich#9785] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On mobile you make a photo selection and from that create a new album. I would love to have this feature on web as well.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
After making a selection, the plus button in the top bar lets you create a new album.

**Comment by erikvanoosten** [comment]
I am going crazy, indeed, now I see the option. But why didn't I see it this morning???
Sorry for the noise!
